README.md

```diff
@@ -255,18 +269,84 @@ The drone will first change its altitude to reach the goal height. It is possibl
+
+The officially supported camera is Intel Realsense D435. We recommend using Firmware version 5.9.13.0. The instructions on how to update the Firmware of the camera can be found [here](https://www.intel.com/content/www/us/en/support/articles/000028171/emerging-technologies/intel-realsense-technology.html)
+
+Pro Tip: Be careful when attaching the camera with a USB3 cable. USB3 might might interfere with GPS and other signals. If possible, always use USB2 cables.
+
+Orther tested camera models are: Intel Realsense D415 and R200.
+
+#### Generating Point-clouds from Depth-maps
+
+In case the point-cloud stream already exists, this step can be skipped.
+
+Assuming there already exists a stream of depth-maps on the ROS-topic <depthmap_topic>, we need to generate a corresponding stream of depth-maps.
+Start by following the instructions from [PX4/disparity_to_point_cloud](https://github.com/PX4/disparity_to_point_cloud).
+Now run the point-cloud generation with the parameters for the camera intrinsics:
 
-It is possible to log the point cloud, the polar histogram, the waypoints and the drone position when simulating the local planner. In order to (dis)enable logging, you need to set the following arguments to `false`/`true` in the `local_planner_depth-camera.launch` launch file:
 ```bash
-<arg name="record_point_cloud" default="true" />
-<arg name="record_histogram"   default="true" />
-<arg name="record_waypoints"   default="true" />
-<arg name="record_position"    default="true" />
+rosrun disparity_to_point_cloud disparity_to_point_cloud_node \
+    fx_:=fx fy_:=fy cx_:=cx cy_:=cy base_line_:=base_line disparity:=<depthmap_topic>
 ```
-You can retrieve the logs in the `~/.ros/` folder of you computer.
 
-## Troubleshooting
+A stream of point-clouds should now be published to */point_cloud*.
+
+### PX4 Autopilot
+
+Parameters to set through QGC:
+* MPC_OBS_AVOID to 1
+* SYS_COMPANION to 57600 or
+
+### Companion Computer
+
+* OS: Ubuntu 16.04 OS or a docker container running Ubuntu 16.04 must be setup (e.g. if using on a Yocto based system). 
+* ROS Kinetic: see [Installation](#installaton)
+* Other Required Components for Intel Realsense:
+  - Librealsense (Realsense SDK). The installation instructions can be found [here](https://github.com/IntelRealSense/librealsense/blob/master/doc/installation.md)
+  - [Librealsense ROS wrappers)](https://github.com/intel-ros/realsense.git)
+
+Tested models for the local planner Intel NUC, Jetson TX2, Intel Atom x7-Z8750 (built-in on Intel Aero RTF drone), for the global planner Odroid.
+
+## Global Planner
+
+The global planner has been so far tested on a Odroid by the development team. 
+
+For more information, read the [Running on Odroid](https://github.com/PX4/avoidance/tree/master/global_planner/resource/odroid) instructions.
+
+## Local Planner
+
+Once the catkin workspace has been built, to run the planner with a Realsense D435 camera launch local_planner_example.launch editing the arguments:
```

Comment:
highlight *local_planner_example.launch*. When you say arguments, I presume you mean that there are command line arguments? if so, then perhaps show an example: